### PR TITLE
Add IFM

### PR DESCRIPTION
**Pull request by misterunknown**
I hope this meets all contributing guidelines.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 entries with syntax error(s).</td>
    </tr>
  </tbody>
</table>

#### Syntax Issues

 Name | Entry
----|----------------------
IFM |   * [IFM](https://github.com/misterunknown/ifm/) - A single script file manager `MIT` `PHP` 
 

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -639,6 +639,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
   * [eXtplorer](https://extplorer.net/) - A PHP and Javascript based File Manager. ([Source Code](https://extplorer.net/projects/extplorer/repository)) `MPLv1.1/GPLv2` `PHP`
   * [goBrowser](https://github.com/xataz/gobrowser) - Simple http file browser. ([Source Code](https://github.com/xataz/gobrowser)) `GPLv3` `Go`
   * [h5ai](https://larsjung.de/h5ai/) - Modern file indexer for HTTP web servers with focus on your files. Directories are displayed in a appealing way and browsing them is enhanced by different views, a breadcrumb and a tree overview. ([Demo](https://larsjung.de/h5ai/demo/), [Source Code](https://github.com/lrsjng/h5ai)) `MIT` `PHP`
+  * [IFM](https://github.com/misterunknown/ifm/) - A single script file manager ([Source Code](https://github.com/misterunknown/ifm/)) `MIT` `PHP`
```
Comment (nodiscc) [inline README.md]: Please remove the source code links as it is the same as the main link

Comment (nodiscc) [inline README.md]: Missing final `.` after the description

**Comment by Kickball** [comment]
@misterunknown at the moment the entry makes no mention of what IFM stands for. I will merge this anyway, please submit another PR if you would like to change the entry to include that description.

**Pull request merged**